Update serverless installation docs (#8116)
DataDog/documentation: content/en/serverless/installation/go.md

go get github.com/DataDog/datadog-lambda-go
```

### Configure the Function

1. Set environment variable `DD_FLUSH_TO_LOG` to `true`.
1. Enable [AWS X-Ray active tracing][3] for your Lambda function.

### Subscribe the Datadog Forwarder to the Log Groups

You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups, in order to send metrics, traces and logs to Datadog.

1. [Install the Datadog Forwarder if you haven't][4].
2. [Ensure the option DdFetchLambdaTags is enabled][5].
3. [Subscribe the Datadog Forwarder to your function's log groups][6].

> ## Quick Start

Reviewer: ```suggestion
## Explore Datadog Serverless Monitoring
```

> After you have configured your function following the steps above, check out the sample code below to get tarted, and view metrics, logs and traces on the [Serverless page][7].

Reviewer: ```suggestion
After you have configured your function following the steps above, check out the sample code below to get started, and view metrics, logs and traces on the [Serverless page][7].
```

> After you have configured your function following the steps above, check out the sample code below to get tarted, and view metrics, logs and traces on the [Serverless page][7].

Reviewer: ```suggestion
After you have configured your function following the steps above, you should be able to view metrics, logs and traces on the [Serverless page][7]. If you need to submit a custom metric, refer to the sample code below:
```
